### ISSUE TITLE
508-defect-2: Long file names overflow small screens

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#54512]
### Point of contact

Josh Kim

### Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

### Details

On a small (e.g. 320 pixel wide) screen, when uploading a long file the upload card will overflow and go off screen

![super_long_file_name_123412345234623562362346234623462346.pdf file name extends off the screen. This also causes the document type select](https://user-images.githubusercontent.com/136959/222539427-85db6c03-198b-46fa-9cfd-8e026ad84c66.png)

### Reproduction steps

1. In staging, log into any of the [HLR users](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/staging-test-accounts-HLR.md)
3. Go to [Supplemental Claims form](https://staging.va.gov/decision-reviews/supplemental-claim/file-supplemental-claim-form-20-0995)
4. Navigate to the evidence upload page (choose yes)
5. Upload a file name with a really long name
6. Shrink browser width to cause the file name to overflow (depends on file name length)

### Proposed solution or next steps

Use CSS to force wrap the long file name

### References, articles, or WCAG support

1.
2.
3.
...

### Type of issue

- [ ] Screenreader
- [ ] Keyboard
- [ ] Focus
- [ ] Headings
- [ ] Color
- [X] Zoom
- [ ] Semantics
- [ ] Axe-core
- [ ] Something else

**Comment by Mottie** [comment]
Ready for testing in staging

**Comment by saderagsdale** [comment]
@joshkimux Hey! Let us know if this one looks good to close.

**Comment by joshkimux** [comment]
<img width="320" alt="Screenshot 2023-03-08 at 4 13 11 PM" src="https://user-images.githubusercontent.com/14154792/223851343-6815493f-0ac8-46b4-8151-ceec797d25e2.png">

look at that lil thing wrap 🌯 ❤️
